From 675ce908a33d16f3b78d3fc741b0ff45790e4770 Mon Sep 17 00:00:00 2001
From: Fabian Vogt <fabian@ritter-vogt.de>
Date: Wed, 17 May 2017 17:05:41 +0200
Subject: Fix compilation with GCC 7

std::bind is part of functional, and GCC 7 requires an explicit include.
---
 src/kleo/enum.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/kleo/enum.cpp b/src/kleo/enum.cpp
index 15ea1fd..829d4fd 100644
--- a/src/kleo/enum.cpp
+++ b/src/kleo/enum.cpp
@@ -33,6 +33,9 @@
 #include "enum.h"
 #include "libkleo_debug.h"
 #include "models/keycache.h"
+
+#include <functional>
+
 #include <KLocalizedString>
 
 #include <gpgme++/key.h>
-- 
cgit v0.11.2

